docs/examples/errorprone/Makefile

```diff
@@ -7,6 +7,12 @@ all:
 	grep -qF "Demo.java:7: warning: [CollectionIncompatibleType] Argument 'i - 1' should not be passed to this method; its type int is not compatible with its collection's type argument Short" Out.txt
 	grep -qF "Demo.java:8: error: [argument.type.incompatible] incompatible argument for parameter arg0 of Set.add." Out.txt
 
+ifeq ($(OS),Windows_NT)
+    GRADLEW_CMD := ..\..\..\gradlew.bat
+else
+    GRADLEW_CMD := ../../../gradlew
+endif
+
 clean:
-	../../../gradlew clean
+	$(GRADLEW_CMD) clean
```

Comment:
How about undoing these changes and just using `- ../../...` linke in the `all` target above?
I assume that running all on Windows similarly fails at the moment.
For the `clean` tasks it is fine to silently not work, e.g. if a program isn't installed.

Reply:
I remembered I did it here [17b2897](https://github.com/eisop/checker-framework/pull/896/commits/17b2897b68fd2a2cd93c63d05356dce145465816) but it did not work. But let me try again.